### ISSUE TITLE
Remove installation warnings

### ISSUE THREAD

**Issue by michaelfarrell76** [readmeio/rdme#69]
I get the following warnings when I install with yarn, the only warnings in our entire monorepo are being emitted by readmeio packages

```
warning rdme > opn@6.0.0: The package has been renamed to `open`
```

**Comment by erunion** [comment]
Resolved in 3.4.5

**Comment by michaelfarrell76** [comment]
I'm getting new warnings when upgrading to 3.4.5

```
warning rdme > oas > base-store > data-store > cache-base > set-value@0.4.3: Critical bug fixed in v3.0.1, please upgrade to the latest version.
warning rdme > oas > base-store > data-store > union-value > set-value@0.4.3: Critical bug fixed in v3.0.1, please upgrade to the latest version.
```

**Comment by erunion** [comment]
@michaelfarrell76 Whoops, I forgot to update our `oas` dependency after fixing that issue there. This should all be resolved in 3.4.6 now.
